client/src/App.js

```diff
@@ -91,7 +92,11 @@ class Board extends Component {
     connection.onmessage = message => {
       if (this._isMounted) {
         let parsed = JSON.parse(message.data);
-        this.setState({ tiles: parsed.tiles})
+        this.setState({
+          tiles: parsed.codies.tiles,
+          playerId: parsed.playerId,
+          isSpymaster: parsed.resetSpymasters ? false : this.state.isSpymaster,
```

Comment:
Is this a javascript thing I don't understand? Can you just do `parsed.resetSpymasters == true`

Reply:
no, just in case someone wanted to send `resetSpymasters = false`, it won't override your existing spymaster status. nothing to do with js. also in javascript you have to always use `===` because javascript is stupid